spongehome.go

```diff
@@ -26,23 +26,65 @@
 package main
 
 import (
-    "github.com/SpongePowered/SpongeHome/controllers"
-    "github.com/go-macaron/pongo2"
-    "gopkg.in/macaron.v1"
+	"fmt"
+	"github.com/SpongePowered/SpongeHome/controllers"
+	"github.com/go-macaron/pongo2"
+	"github.com/sethvargo/go-fastly"
+	"gopkg.in/macaron.v1"
+	"os"
+	"time"
 )
 
+func clearfastly() {
+	//Clear Fastly cache
```

Comment:
the space is crucial :P 
unfortunately this goes for the rest of the comments too